### PR TITLE
Fix smhi data in unittests

### PR DESCRIPTION
**Pull request by mathiasb**
Updated the test case test_get_n_prep_data_from_smhi to just check that the right columns are in the datasource, and that the values are all positive real numbers

### CODE REVIEW

File: data/tests/test_emission_calculations.py [hunk 1]
```diff
@@ -22,12 +22,18 @@ class TestEmissionCalculations(unittest.TestCase):
...
-        pd.testing.assert_frame_equal(df_result, df_expected, check_dtype=False)
+        result_columns = df_result.columns.to_list()[4:] # Skip the first 4 columns which is the 'index'
+        expected_columns = [1990, 2000, 2005, 2010, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022]
+
+        # Check that the expected columns are in the dataframe
+        assert result_columns == expected_columns
+
+        # Each of the column values should all be greater than 0.0
+        for col in expected_columns:
+            assert (df_result[col] > 0.0).all() == True
```
Comment (elvbom) [inline data/tests/test_emission_calculations.py:36]: Wondering if '==' could be replaced with 'is' for better clarity.

Comment (Greenheart) [reply data/tests/test_emission_calculations.py:36]: To me both are clear, and it doesn't matter as long as we're consistent :)

Related: Could we automatically format python code just like `prettier` does for JS/TS?

Comment (mathiasb) [reply data/tests/test_emission_calculations.py:36]: Either way for me. Happy to change to 'is' if that is the preferred way. Let me know which way!

Comment (mathiasb) [reply data/tests/test_emission_calculations.py:36]: @Greenheart, there are numerous code formatters and linters for Python. This article presents a nice smorgasbord to meditate over : https://inventwithpython.com/blog/2022/11/19/python-linter-comparison-2022-pylint-vs-pyflakes-vs-flake8-vs-autopep8-vs-bandit-vs-prospector-vs-pylama-vs-pyroma-vs-black-vs-mypy-vs-radon-vs-mccabe/

Comment (elvbom) [reply data/tests/test_emission_calculations.py:36]: 'is' is indeed preferred :)

**Pull request merged**